test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -895,5 +910,21 @@ private void VerifyAndCleanup(string filePath)
                 File.Delete(filePath);
             }
         }
+
+        /// <summary>
+        /// Verify expected and actual DeploymentOptions BooleanOptionsDictionary values
+        /// </summary>
+        /// <param name="expectedBooleanOptionsDictionary"></param>
+        /// <param name="actualBooleanOptionsDictionary"></param>
+        public void VerifyExpectedAndActualBooleanOptionsDictionary(Dictionary<string, DeploymentOptionProperty<bool>> expectedBooleanOptionsDictionary, Dictionary<string, DeploymentOptionProperty<bool>> actualBooleanOptionsDictionary)
+        {
+            foreach (KeyValuePair<string, DeploymentOptionProperty<bool>> optionRow in expectedBooleanOptionsDictionary)
+            {
+                var expectedValue = optionRow.Value.Value;
+                var actualValue = actualBooleanOptionsDictionary[optionRow.Key].Value;
+
+                Assert.That(actualValue, Is.EqualTo(expectedValue), $"Actual Property from Service is not equal to default property for {optionRow.Key}, Actual value: {actualValue} and Expected value: {expectedValue}");
```

Comment:
The error message will already give the actual and expected value (that's part of the reason why using these assertions is better). So you can simplify this message to just the first part of the sentence.